sdk/tests/conftest.py

```diff
@@ -380,7 +432,103 @@ def router_version(
         timeout="100ms",
         log_config=log_config,
         ensembler=ensembler,
-        monitoring_url="https://lookhere.io/"
+        monitoring_url="https://lookhere.io/",
+        enricher=generic_enricher
+    )
+
+
+@pytest.fixture
+def generic_router_config():
+    return turing.router.config.router_config.RouterConfig(
+        environment_name="id-dev",
+        name="router_1",
+        routes=[
+            turing.router.config.route.Route(
+                id="model-a",
+                endpoint="http://predict-this.io/model-a",
+                timeout="100ms"
+            ),
+            turing.router.config.route.Route(
+                id="model-b",
+                endpoint="http://predict-this.io/model-b",
+                timeout="100ms"
+            )
+        ],
+        rules=None,
+        default_route_id="test",
+        experiment_engine=turing.experiment_config.ExperimentConfig(
+            type="xp",
+            config={
+                'variables':
+                        [
+                            {'name': 'order_id', 'field': 'fdsv', 'field_source': 'header'},
+                            {'name': 'country_code', 'field': 'dcsd', 'field_source': 'header'},
+                            {'name': 'latitude', 'field': 'd', 'field_source': 'header'},
+                            {'name': 'longitude', 'field': 'sdSDa', 'field_source': 'header'}
+                        ],
+                'project_id': 102
+            }
+        ),
+        resource_request=turing.router.config.resource_request.ResourceRequest(
```

Comment:
Ok sounds good! I've just refactored all the tests in the router module to use classes imported using import statements for the SDK classes so the code should a lot less verbose now. The autogenerated classes are still used via imported modules, i.e. `turing.generated.models.[class_name]` because they usually have the same names as those of the SDK classes...

Another alternative is to import them (the autogenerated classes) with aliases, e.g. ApiResourceRequest or GenResourceRequest, but I'm not sure if that'll be confusing/too wordy.